modules/embargoes_log_views/embargoes_log_views.info.yml

```
@@ -3,5 +3,5 @@ description: 'Provides views for embargo logs.'
 type: module
 core_version_requirement: ^8 || ^9
 dependencies:
-  - embargoes
-  - views
+  - fsulib:embargoes
```

This is off... is more `module:(sub)module` name... `drupal:views` is appropriate, as `views` is core Drupal module.

```suggestion
  - embargoes:embargoes
```

---

Made suggested change.